openhsi/cameras.py

```diff
@@ -331,4 +331,50 @@ def get_temp(self) -> float:
     def get_mac(self)-> str:
         return ':'.join(['{}{}'.format(a, b)
                          for a, b
-                         in zip(*[iter('{:012x}'.format(cam.deviceSettings['GevMACAddress'].value))]*2)])
\ No newline at end of file
+                         in zip(*[iter('{:012x}'.format(cam.deviceSettings['GevMACAddress'].value))]*2)])
+
+# Cell
+from typing import Tuple
+
+@delegates()
+class FlirCamera(OpenHSI):
+    """Interface for FLIR camera"""
+
+    def __init__(self, win_resolution:Tuple[int,int] = None, win_offset:Tuple[int,int] = None, exposure_us:float = 4_000, **kwargs):
```

Comment:
exposure, win resolution and binning come from setting file, should make consistent with lucid camera class.

Reply:
Agreed. Do you have a more up to date settings json file I can match to? 
![image](https://user-images.githubusercontent.com/46410481/149458235-ea267557-e056-439a-b8a4-8d0c05b779d8.png)
Should the default win_resolution be non-zero here?